src/main/java/io/odpf/kafkaclient/console/Console.java

```diff
@@ -1,7 +1,23 @@
 package io.odpf.kafkaclient.console;
 
+import io.odpf.kafkaclient.Client;
+import io.odpf.kafkaclient.ClientFactory;
+import io.odpf.kafkaclient.consumer.ConsumerFactory;
+import io.odpf.kafkaclient.producer.ProducerFactory;
+
+import java.io.File;
+import java.io.IOException;
+
 public class Console {
-    public static void main(String... args) {
 
+    public static void main(String... args) throws IOException {
+        File propertiesFile = new File(args[1]);
+        ClientFactory clientFactory = (args[0].equals("producer")) ?
```

Comment:
what if my agr is not producer or consumer ? Handle edge-cases properly.

Reply:
This could have been an ENUM no.

Reply:
Handled all edge cases in a newer commit